### PR TITLE
Update command in codegen in JS command

### PR DESCRIPTION
**Pull request by MaryGao**
Update command in codegen in JS command

### CODE REVIEW

File: tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ async function automationGenerateInPipeline(inputJsonPath: string, outputJsonPat
     const inputJson = JSON.parse(fs.readFileSync(inputJsonPath, {encoding: 'utf-8'}));
     const specFolder: string = inputJson['specFolder'];
     const readmeFiles: string[] | string | undefined = inputJson['relatedReadmeMdFiles']? inputJson['relatedReadmeMdFiles']: inputJson['relatedReadmeMdFile'];
-    const cadlProjectFolder: string[] | string | undefined = inputJson['relatedCadlProjectFolder'];
+    const cadlProjectFolder: string[] | string | undefined = inputJson['relatedTypespecProjectFolder'];
```
Comment (lirenhe) [inline tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts:17]: Please also update 'cadlProjectFolder'

File: tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts [hunk 2]
```diff
@@ -65,7 +65,7 @@ async function automationGenerateInPipeline(inputJsonPath: string, outputJsonPat
             cadlProject: cadlProject,
             autorestConfig,
             use: use,
-            cadlEmitter: !!cadlEmitter? cadlEmitter : `@azure-tools/cadl-typescript`,
+            cadlEmitter: !!cadlEmitter? cadlEmitter : `@azure-tools/typespec-ts`,
```
Comment (lirenhe) [inline tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts:68]: Shall we still use 'cadlEmitter'?

Comment (lirenhe) [reply tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts:68]: It would be good to do a full search for all cadl in the source code.

Comment (MaryGao) [reply tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts:68]: Sure, let me update the name accordingly

File: tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts (hunk 1 above)
Comment (benbp) [reply tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts:17]: Also the TypeSpec capitalization should be `relatedTypeSpecProjectFolder`.

Comment (raych1) [reply tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts:17]: @MaryGao , please use capitalization. I made the change in sdk automation contract schema.

File: tools/js-sdk-release-tools/src/utils/getOutputPackageInfo.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ export function getOutputPackageInfo(runningEnvironment: RunningEnvironment | un
             "result": "succeeded"
         };
         if (cadlProject) {
-            outputPackageInfo['cadlProject'] = [cadlProject];
+            outputPackageInfo['typeSpecProject'] = [cadlProject];
```
Comment (benbp) [inline tools/js-sdk-release-tools/src/utils/getOutputPackageInfo.ts]:
```suggestion
            outputPackageInfo['typespecProject'] = [cadlProject];
```

**Review by raych1 — APPROVED** [review]

File: tools/js-sdk-release-tools/src/utils/getOutputPackageInfo.ts [hunk 4]
```diff
@@ -16,8 +16,8 @@ export function getOutputPackageInfo(runningEnvironment: RunningEnvironment | un
             "artifacts": [],
             "result": "succeeded"
         };
-        if (cadlProject) {
-            outputPackageInfo['cadlProject'] = [cadlProject];
+        if (typeSpecProject) {
+            outputPackageInfo['typeSpecProject'] = [typeSpecProject];
```
Comment (raych1) [inline tools/js-sdk-release-tools/src/utils/getOutputPackageInfo.ts:20]: This is the naming convention used in other places: either `typespec` or `TypeSpec`. So, please follow it. I just made the changes to the contract in SDK automation framework.

**Review by raych1 — CHANGES_REQUESTED** [review]
Request naming change.

**Review by raych1 — APPROVED** [review]

**Pull request merged**